backend/operations_scanner_test.go

```diff
@@ -468,6 +469,7 @@ func TestConvertClusterStatus(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		var operationsScanner *OperationsScanner
```

Comment:
@mbarnes, I noticed the `OperationsScanner` has a `clusterService` field of type `ocm.ClusterServiceClient`, is that intentional?
To use the mock and add more test coverage, we'll have to change it to the `ClusterServiceClientSpec` interface. I could handle that in a separate PR to keep this one concise and add tests later.

Reply:
Good catch, that is in fact _not_ intentional.  I guess the existing tests weren't mocking any CS calls.  Please fix however you think is best.

Reply:
I created this ticket to track the changes https://issues.redhat.com/browse/ARO-17009, will do in a follow-up PR.